### ISSUE TITLE
Add parentEventID for collection objs from historical expeditions on wikidata

### ISSUE THREAD

**Issue by rukayaj** [gbif-norway/helpdesk#162]
Check the expeditions working group slides presented by @dagendresen to get a list of the objects on the Tristan da Cunha cruise. Publish them to the annotation API https://annotater.svc.gbif.no/ or to the data published on the ipt? I guess nobody will use it and it will not be noticed on the annotater so we should publish to GBIF.

**Comment by dagendresen** [comment]
Yes my thought on the Annotator is for (at least some of) the annotations to be merged into the data pipelines onto the IPT. MANY datatypes will not have any support inside the collection databases (CMS). So bringing back to the source is thus in practice impossible.
